exchanges/btcmarkets/btcmarkets.go

```diff
@@ -4,41 +4,49 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"io"
 	"net/http"
 	"net/url"
+	"strconv"
 	"strings"
+	"time"
 
 	"github.com/thrasher-corp/gocryptotrader/common"
 	"github.com/thrasher-corp/gocryptotrader/common/crypto"
 	"github.com/thrasher-corp/gocryptotrader/currency"
 	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
 	"github.com/thrasher-corp/gocryptotrader/exchanges/websocket/wshandler"
-	log "github.com/thrasher-corp/gocryptotrader/logger"
 )
 
 const (
-	btcMarketsAPIURL            = "https://api.btcmarkets.net"
-	btcMarketsAPIVersion        = "0"
-	btcMarketsAccountBalance    = "/account/balance"
-	btcMarketsTradingFee        = "/account/%s/%s/tradingfee"
-	btcMarketsOrderCreate       = "/order/create"
-	btcMarketsOrderCancel       = "/order/cancel"
-	btcMarketsOrderHistory      = "/order/history"
-	btcMarketsOrderOpen         = "/order/open"
-	btcMarketsOrderTradeHistory = "/order/trade/history"
-	btcMarketsOrderDetail       = "/order/detail"
-	btcMarketsWithdrawCrypto    = "/fundtransfer/withdrawCrypto"
-	btcMarketsWithdrawAud       = "/fundtransfer/withdrawEFT"
-
-	// Status Values
-	orderStatusNew                = "New"
-	orderStatusPlaced             = "Placed"
-	orderStatusFailed             = "Failed"
-	orderStatusError              = "Error"
-	orderStatusCancelled          = "Cancelled"
-	orderStatusPartiallyCancelled = "Partially Cancelled"
-	orderStatusFullyMatched       = "Fully Matched"
-	orderStatusPartiallyMatched   = "Partially Matched"
+	btcMarketsAPIURL     = "https://api.btcmarkets.net"
+	btcMarketsAPIVersion = "/v3"
+
+	// UnAuthenticated EPs
+	btcMarketsAllMarkets         = "/markets/"
+	btcMarketsGetTicker          = "/ticker/"
+	btcMarketsGetTrades          = "/trades?"
+	btcMarketOrderBooks          = "/orderbook?"
+	btcMarketsCandles            = "/candles?"
+	btcMarketsTickers            = "/tickers?"
+	btcMarketsMultipleOrderbooks = "/orderbooks?"
+	btcMarketsGetTime            = "/time"
+	btcMarketsUnauthPath         = btcMarketsAPIURL + btcMarketsAPIVersion + btcMarketsAllMarkets
+
+	// Authenticated EPs
+	btcMarketsAccountBalance = "/accounts/me/balances"
+	btcMarketsTradingFees    = "/accounts/me/trading-fees"
+	btcMarketsTransactions   = "/accounts/me/transactions"
+	btcMarketsOrders         = "/orders"
+	btcMarketsTradeHistory   = "/trades"
+	btcMarketsWithdrawals    = "/withdrawals"
+	btcMarketsDeposits       = "/deposits"
+	btcMarketsTransfers      = "/transfers"
+	btcMarketsAddresses      = "/addresses"
+	btcMarketsWithdrawalFees = "/withdrawal-fees"
```

Comment:
Move this to the Unauthenticated Eps section

Reply:
those are auth eps btw

Reply:
any particular reason for moving them to unauth?